site/src/App/foundations/layout/layout.tsx

```diff
@@ -0,0 +1,512 @@
+          {spaceScale.map(space => (
+            <Columns space="xsmall" alignY="center">
+              <Column width="content">
+                <Box style={{ width: 100 }}>
+                  <Text baseline={false}>{space}</Text>
+                </Box>
+              </Column>
+              <Column>
+                <Box display="flex">
+                  <Box
+                    background="formAccent"
+                    paddingLeft={space}
+                    paddingTop="medium"
+                  />
+                </Box>
+              </Column>
+            </Columns>
+          ))}
+        </Stack>
+
+        <Text tone="secondary" size="small">
+          Note: The exact pixel values in this scale may vary betweeen themes.
+        </Text>
+
+        <Divider />
+
+        <Heading level="3">Box</Heading>
+        <Text>
+          Box is the most low-level layout component provided by Braid. Its job
+          is to render an individual element on the screen.
+        </Text>
+        <Text>
+          In terms of page layout, Box most notably provides a set of padding
+          options which can be used to create container elements with internal
+          spacing.
+        </Text>
+        <BulletList>
+          <Bullet>padding</Bullet>
+          <Bullet>paddingX</Bullet>
+          <Bullet>paddingY</Bullet>
+          <Bullet>paddingTop</Bullet>
+          <Bullet>paddingBottom</Bullet>
+          <Bullet>paddingLeft</Bullet>
+          <Bullet>paddingRight</Bullet>
+        </BulletList>
+        <Text>These options accept a value from our white space scale.</Text>
+        <Text>
+          For example, if you wanted to create a container element with small
+          spacing on all sides
+        </Text>
+        <Code>
+          <Box padding="small">
+            <Text>Lorem ipsum dolor sit amet.</Text>
+          </Box>
+        </Code>
+        <Text>
+          These also support the ‘responsive props’ format which allows you to
+          specify an array of values for different screen sizes.
+        </Text>
+        <Text>
+          For example, if you wanted small spacing on mobile but medium spacing
+          from tablet upwards:
+        </Text>
+        <Code>
+          <Box padding={['small', 'medium']}>
+            <Text>Lorem ipsum dolor sit amet.</Text>
+          </Box>
+        </Code>
+        <Text>
+          If required, you’re also able to specify a different value for desktop
+          screens.
+        </Text>
+        <Text>
+          For example, if you wanted to set the previous example’s spacing to
+          large on desktop:
+        </Text>
+        <Code>
+          <Box padding={['small', 'medium', 'large']}>
+            <Text>Lorem ipsum dolor sit amet.</Text>
+          </Box>
+        </Code>
+
+        <Divider />
+
+        <Heading level="3">Card</Heading>
+        <Text>
+          Rather than nesting content in arbitrary Box elements, you may prefer
```

Comment:
This Card section seems weirdly perfunctory compared to the other sections - do you need to flesh it out, or is the implication here that it's not an important concern for layout?

Reply:
All the examples after it use cards so I wanted to introduce it. There's really not much to cards currently, so there's not a lot to say 😄

Reply:
I think it would be good to contextualise its role a *little* bit more, but otherwise all good.